docker/components/sitl-server/Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN ["/bin/bash", "-c", " \
         DONT_RUN=1 make posix_sitl_default gazebo \
 "]
 
-RUN echo "param set MAV_BROADCAST 1" >> ${FIRMWARE_DIR}/posix-configs/SITL/init/ekf2/iris
+RUN echo "param set MAV_BROADCAST 1" >> ${FIRMWARE_DIR}/ROMFS/px4fmu_common/init.d-posix/10016_iris}
```

Comment:
it could be `1015_iris_obs_avoid` such that `COM_OBS_AVOID` is enabled by default

Reply:
Hm okay, I'll revisit this once I got it working in its entirety :)
Really not sure know...